### ISSUE TITLE
Feature Request: detect models in folder (and subfolders)

### ISSUE THREAD

**Issue by mkellerman** [alexrozanski/LlamaChat#27]
Would be nice to maybe pickup the sha of all the models in a folder (recursively), and auto-import all the models in the tool.

**Comment by alexrozanski** [comment]
Yes, this would be nice! Probably won't be getting to it any time soon but will consider this after the upcoming changes to support more and newer models
